lib/src/test/java/com/auth0/android/jwt/ClaimImplTest.java

```diff
@@ -241,4 +241,27 @@ public void shouldThrowIfListClassMismatch() throws Exception {
         exception.expect(DecodeException.class);
         claim.asList(UserPojo.class);
     }
+
+    @Test
+    public void shouldGetSubClaimsObject() throws Exception {
+        String username = "Sabin";
+
+        SubClaimsPojo data = new SubClaimsPojo();
```

Comment:
No need to create a new class. Use the `UserPojo` that already exists.

Reply:
Deleted SubClaimsPojo class and used UserPojo class instead.